uxarray/_exodus.py

```diff
@@ -0,0 +1,223 @@
+                    # nothing specified set to m
+                    "units": "deg",
+                })
+        elif key == "coordx":
+            self.in_ds["Mesh2_node_x"] = xr.DataArray(
+                data=ext_ds.coordx,
+                dims=["nMesh2_node"],
+                attrs={
+                    # nothing specified set to spherical
+                    "standard_name": "spherical",
+                    "long_name": ext_ds.title,
+                    # nothing specified set to m
+                    "units": "deg",
+                })
+        elif key == "coordy":
+            self.in_ds["Mesh2_node_y"] = xr.DataArray(
+                data=ext_ds.coordx,
+                dims=["nMesh2_node"],
+                attrs={
+                    # nothing specified set to spherical
+                    "standard_name": "spherical",
+                    "long_name": ext_ds.title,
+                    # nothing specified set to m
+                    "units": "deg",
+                })
+        elif key == "coordz":
+            self.in_ds["Mesh2_node_z"] = xr.DataArray(
+                data=ext_ds.coordx,
+                dims=["nMesh2_node"],
+                attrs={
+                    # nothing specified set to spherical
+                    "standard_name": "spherical",
+                    "long_name": ext_ds.title,
+                    # nothing specified set to m
+                    "units": "deg",
+                })
+        elif key == "connect1":
+            for k, v in value.attrs.items():
+                if k == "elem_type":
+                    etype = v
+            self.in_ds["Mesh2_face_nodes"] = xr.DataArray(
+                data=(ext_ds.connect1[:] - 1),  #Note: -1
+                dims=["nMesh2_face", "nMaxMesh2_face_nodes"],
+                attrs={
+                    "cf_role": "face_node_connectivity",
+                    "_FillValue": -1,
+                    "start_index":
+                        0  #NOTE: This might cause an error if numbering has holes
+                })
+    print("Finished reading exodus file.")
+
+
+def write_exodus(self, outfile):
+    # Note this is 1-based unlike native Mesh2 construct
+    print("Writing exodus file: ", outfile)
+
+    self.exo_ds = xr.Dataset()
+
+    path = PurePath(outfile)
+    out_filename = path.name
+
+    now = datetime.now()
+    date = now.strftime("%m/%d/%Y")
+    time = now.strftime("%H:%M:%S")
+
+    title = f"uxarray(" + str(out_filename) + ")" + date + ": " + time
+    fp_word = np.int32(8)
+    version = np.float32(5.0)
+    api_version = np.float32(5.0)
+    self.exo_ds.attrs = {
+        "api_version": api_version,
+        "version": version,
+        "floating_point_word_size": fp_word,
+        "file_size": 0,
+        "title": title
+    }
+
+    self.exo_ds["time_whole"] = xr.DataArray(data=[], dims=["time_step"])
+
+    # qa_records
+    qa_records = [["uxarray"], ["1.0"], [date], [time]]
```

Comment:
Curious about the hardcoded 1.0 value here.

Reply:
Added a variable for the same resolving.